.github/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+# Description
+
+<!-- Please include a summary of the change. Please also include relevant motivation and context. -->
+
+<!-- Please link the issue which will be fixed (if any) here: -->
+Fixes # (issue)
+
+## Type of change
+
+<!-- Please delete options that are not relevant. -->
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+## Checklist:
+
+- [ ] My code follows the style guidelines of this project
+- [ ] I have performed a self-review of my own code
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have made corresponding changes to the documentation
+- [ ] I have added tests that prove my fix is effective or that my feature works
+- [ ] New and existing unit tests pass locally with my changes
```

Comment:
When a new feature is introduced, it must be documented. Otherwise nobody knows it does exist ;-)

Proposal:
`[  ] I have documented the new feature in README.md or ENVIRONMENT.md or the Wiki`

Reply:
I have reformulated the checklist as imho every PR should contain the whole list and every point should be checked by the author. Therefore it isn't suitable to add a point which  is only valid for new feature PRs.

Are you okay with that?